getting_hired/preparing_for_job_search/project_portfolio.md

```diff
@@ -13,8 +13,8 @@ Don't underestimate the importance of high quality documentation on your project
 Remember to write some documentation for both technical and non-technical audience.  A potential employer should be able to follow your documentation, regardless of their technical background.
 
 * [A Beginner’s Guide to Writing Documentation](http://www.writethedocs.org/guide/writing/beginners-guide-to-docs/)
-
-### Show Designs
+* If you're looking for other projects to present in your portfolio, consider participating in the open source community. It's a great way to practice the skills you've developed from our curriculum and gain real-world experience. Check out [this article](https://dev.to/opensauced/how-i-got-hired-contributing-to-open-source-projects-546i) to learn how open source can help you in your job search. 
+  
```

Comment:
Can you explain why you removed the "Show Designs" heading?

In addition, can you explain why you put this in the portfolio project lesson, as opposed to the "What You Can Do to Prepare" lesson that you pointed to when you made the issue?

Reply:
> Can you explain why you removed the "Show Designs" heading?
> 
> In addition, can you explain why you put this in the portfolio project lesson, as opposed to the "What You Can Do to Prepare" lesson that you pointed to when you made the issue?

Hi @rlmoser99! :) Here's a new version of my PR: https://github.com/TheOdinProject/curriculum/pull/25970